src/plugins/discover/public/application/components/top_nav/open_search_panel.tsx

```diff
@@ -92,6 +92,7 @@ export function OpenSearchPanel({ onClose, makeUrl }: Props) {
             },
           ]}
           onChoose={(id) => {
+            data.query.filterManager.setFilters([]); // resetting the filters
```

Comment:
Lets add a test to make sure we do this. Also does this clear global filters?

Reply:
Yes. This does clear the global filters.